src/process.cr

```diff
@@ -46,6 +46,28 @@ class Process
     Crystal::System::Process.signal(pid, signal.value)
   end
 
+  # Installs *handler* as the new handler for interrupt requests. Removes any
+  # previously set interrupt handler.
+  #
+  # The handler is executed on a fresh fiber every time an interrupt occurs.
+  #
+  # * On Unix-like systems, this traps SIGINT.
+  # * On Windows, this captures Ctrl+C and Ctrl+Break signals sent to a console
+  #   application.
+  def self.on_interrupt(&handler : ->) : Nil
+    Crystal::System::Process.on_interrupt(&handler)
+  end
+
+  # Ignores all interrupt signals if *ignore* is true, or restores the default
+  # interrupt handler if *ignore* is false. Removes any custom interrupt
+  # handler set with `on_interrupt(&handler)`.
+  #
+  # * On Windows, interrupts generated by Ctrl+Break cannot be ignored in this
+  # way.
+  def self.on_interrupt(*, ignore : Bool) : Nil
```

Comment:
I fear this method signature might not be very user friendly. Especially `ignore: false` to mean `reset` is not very intuitive. 

As an alternative, perhaps this could be two methods: `Process.ignore_interrupt!` and `Process.reset_interrupt!`?